geschaeftsordnung-vollversammlung.md

```diff
@@ -0,0 +1,74 @@
+
+# Geschäftsordnung der Vollversammlung der Fachschaft Digital Engineering
+
+Diese Geschäftsordnung wurde am xx.xx.xxxx von der Vollversammlung beschlossen.
+
+## § 1 Geltungsbereich
+
+Die Geschäftsordnung gilt für die Vollversammlung der Fachschaft Digital Engineering an der Universität Potsdam.
+
+## § 2 Tagesordnung
+
+(1) Die Vollversammlung folgt einer Tagesordnung.
+
+(2) Die Tagesordnung muss Teil der fristgerechten Einladung einer Vollversammlung sein.
+
+(3) Anderungsanträge zur Tagesordnung einer ordentlichen Vollversammlung können bis zu 14 Tage vor der Vollversammlung bei den Einladenden gestellt werden.
+
+(4) 7 Tage vor einer ordentlichen Vollversammlung muss die finale Tagesordnung auf dem gleichen Weg wie die Einladung zur Vollversammlung bekanntgegeben werden.
+
+## § 3 Versammlungsleitung
+
+(1) Die Vollversammlung wird von den Einladenden geleitet.
+
+(2) Die Versammlungsleitung kann für einzelne Tagesordnungspunkte an andere Mitglieder der Fachschaft übergeben werden.
+
+(3) Die Versammlungsleitung kann bei Bedarf eine Redeliste führen und / oder die Redezeit begrenzen.
+
+(4) Die Versammlungsleitung entscheidet über die explizite Zählung und Protokollierung von Stimmen.
+
+## § 4 Protokolle
+
+(1) Die Vollversammlung dokumentiert ihre Beschlüsse in einem Protokoll.
+
+(2) Die Versammlungsleitung beauftragt ein Mitglied der Fachschaft mit der Protokollführung.
+
+(3) Das Protokoll wird der Fachschaft spätestens 7 Tage nach der Vollversammlung zugänglich gemacht.
+
+## $ 5 Anträge
+
+Anträge können vor der Behandlung durch die antragstellende Person zurückgezogen werden.
+
+## § 6 Änderungsanträge
+
+(1) Änderungsanträge sind Anträge zur Änderung der Tagesordnung oder zur Änderung von Anträgen.
+
+(2) Änderungsanträge werden vor den jeweiligen Anträgen behandelt.
+
+(3) Änderungsanträge können vor der Vollversammlung schriftlich bekanntgegeben und begründet werden.
+
+## § 7 Anträge zur Geschäftsordnung
+
+(1) Anträge zur Geschäftsordnung werden durch das Heben beider Arme oder anderweitig eindeutig angezeigt.
```

Comment:
Warum die "anderweitig eindeutig angezeigt" Regelung? Ich denke das führt im Zweifel nur zu Verwirrung

Reply:
Damit man auch mit dem Arm wedeln und "ich hab einen Antrag zur Geschäftsordnung" sagen kann

Reply:
Ja, aber das ist halt behindert.

Reply:
Ich find's auch behindert, wenn einer den Arm hebt und sagt "Ich hab einen GO-Antrag" und man dann sagt: "Du musst aber erst beide Arme heben, sonst gilt es nicht"

Reply:
Achso, jetzt versteh ich das. Dann von mir aus :D

Reply:
Finde die Argumentation gut, aber die Formulierung unglücklich. Würde in der zweiten Häfte eher sowas sagen wie "können auch anders angezeigt werden, wenn die Intention für die Versammlungsleitung eindeutig erkennbar ist".
StuPa-GO hat da bspw: "werden durch das Heben beider Hände oder durch Zuruf angezeigt"